examples/inverse/psf_ctf_vertices.py

```diff
@@ -92,17 +94,50 @@
 # %%
 # CTF:
 
-brain_ctf = stc_ctf.plot('sample', 'inflated', 'lh', subjects_dir=subjects_dir)
+brain_ctf = stc_ctf.plot(
+    'sample', 'inflated', 'lh', views='ven', subjects_dir=subjects_dir)
 brain_ctf.add_text(0.1, 0.9, 'sLORETA CTF', 'title', font_size=16)
-brain_ctf.show_view('ventral')
 brain_ctf.add_foci(verttrue, coords_as_verts=True, scale_factor=1., hemi='lh',
                    color='green')
 
 # Maximum of CTF
 brain_ctf.add_foci(vert_max_ctf, coords_as_verts=True, scale_factor=1.,
                    hemi='lh', color='black')
 
-
 # %%
 # The green spheres indicate the true source location, and the black
 # spheres the maximum of the distribution.
+#
+# Volumetric source estimates
+# ---------------------------
+# We can do these same operations for volumetric source estimates:
+
+fname_fwd_vol = meg_path / 'sample_audvis-meg-vol-7-fwd.fif'
+forward_vol = mne.read_forward_solution(fname_fwd_vol)
+inverse_operator_vol = mne.minimum_norm.make_inverse_operator(
+    info=evoked.info, forward=forward_vol, noise_cov=noise_cov)
+rm_lor_vol = make_inverse_resolution_matrix(
+    forward_vol, inverse_operator_vol, method='sLORETA', lambda2=lambda2)
+this_s = forward_vol['src'][0]
+sources_vol = [np.argmin(np.linalg.norm(  # closest to the surface one
+    forward['src'][0]['rr'][forward['src'][0]['vertno'][sources[0]]] -
+    this_s['rr'][this_s['vertno']], axis=1))]
+stc_psf_vol = get_point_spread(
+    rm_lor_vol, forward_vol['src'], sources_vol, norm=True)
+vertno_vol = this_s['vertno']
+max_vert_idx, _ = np.unravel_index(
+    stc_psf_vol.data.argmax(), stc_psf_vol.data.shape)
+head_mri_t = mne.transforms.invert_transform(forward_vol['mri_head_t'])
+loc_true = 1000 * mne.transforms.apply_trans(
+    head_mri_t, this_s['rr'][vertno_vol[sources_vol]])
+loc_max_psf = 1000 * mne.transforms.apply_trans(
+    head_mri_t, this_s['rr'][vertno_vol[max_vert_idx]])
```

Comment:
this code is pretty hairy for new comers. Can we simplify?
one way to do it is to:
- compute stc_psf_vol just after rm_lor_vol
- maybe plot in head coordinate system to avoid the transform here and only plot the peak of the true location?

Reply:
In theory doing `brain.add_foci(idx, coords_as_verts=True)` should work actually. I tried it and it did not work immediately, but I can look into the issue and fix it here

Reply:
done @agramfort, turns out I was just using `add_foci(..., coords_as_verts=True)` incorrectly. It's much simpler now